bool lookLeft
bool lookRight
bool lookUp
bool lookDown
bool home
int32 step